ngsa/src/main/java/com/cse/ngsa/app/NgsaJavaApplication.java

```diff
@@ -18,7 +18,7 @@
   */
   public static void main(String[] args) {
     
-    CommonUtils.handleCliOptions(args);
     SpringApplication.run(NgsaJavaApplication.class, args);
+    CommonUtils.handleCliOptions(args);
```

Comment:
This order is interesting. Is there a chance that the webapp will run before some of the cli options are handled (such as the burst headers)?